srcpkgs/gnupg/gnupg2-scdaemon.INSTALL.msg

```diff
@@ -0,0 +1,5 @@
+GnuPG 2.3 changes the behaviour of scdaemon. While GnuPG 2.2 had a fallback to
+PC/SC when the internal CCID driver didn't work, the fallback now has to be
+explicitly configured by disabling the internal CCID driver. If you've not
+explicitly enabled pcscd, you shouldn't need to do anything. For more details,
+see https://docs.voidlinux.org/config/gnupg.html
```

Comment:
make this `./files/README.voidlinux` and `vdoc` it

Reply:
but then people aren't informed why their setup is breaking when installing the update that breaks their setup.. I think including this as an INSTALL.msg is good. If you want I can link it into ./files/README.voidlinux and vdoc it in addition to that.